### ISSUE TITLE
Error when installing setup.py

### ISSUE THREAD

**Issue by Pante9** [edwardoughton/pysim5g#35]
Based on the instructions given I installed fiona, shapely etc, and now I' m trying to install pysim5g permanently with
**python setup.py develop**
Running the above, or the below
**python setup.py install**
I get the following error.

Any idea on how to fix it?

![error](https://user-images.githubusercontent.com/61228261/113048011-b873d980-91aa-11eb-9529-ca221ad5b5e4.png)

**Comment by Pante9** [comment]
> Hi @Pante9 - it looks like a syntax error from the `f"string"` in the last line there.
> 
> Can you try upgrading the python in your conda environment to 3.7? Try running:
> 
> ```
> conda activate pysim5g
> conda install python=3.7
> ```
> 
> Then try again.
> 
> If that fixes it, we should change the README instructions, which use 3.5.
-------------------------------------------------------------------------------------
Thanks for your time @tomalrussell 

I tried with python 3.7 and got the following error this time



![newnew](https://user-images.githubusercontent.com/61228261/113175584-41931b00-9254-11eb-83dc-28cf1a4c563c.png)

**Comment by tomalrussell** [comment]
Okay - the end of the error message there should be some help - are you running from a ZIP download of the code?

Two different things you could try:
- clone the repository using `git clone https://github.com/edwardoughton/pysim5g.git` on the command line or use an application like GitHub Desktop, and then run `python setup.py develop` from that folder.
- run `pip install git+https://github.com/edwardoughton/pysim5g.git#egg=pysim5g` to install the package directly from GitHub - this should work if you just want to run scripts using pysim5g, but is not so good if you want to make changes to the source code

**Comment by Pante9** [comment]
Yes, I downloaded the code as a zip file and extracted it to a folder into the used path.
After cloning successfully the repository, 
![1a](https://user-images.githubusercontent.com/61228261/113210907-74053e00-927d-11eb-84c2-5e7d1e4aeec9.png)
I got the same error about 'console scripts'.

Same thing when running pip install git+https://github.com/edwardoughton/pysim5g.git#egg=pysim5g
![1b](https://user-images.githubusercontent.com/61228261/113211125-bcbcf700-927d-11eb-92e5-4ae312652938.png)

**Comment by Pante9** [comment]
![prob](https://user-images.githubusercontent.com/61228261/113211418-1de4ca80-927e-11eb-9654-95a9a20c695a.png)
Tried already to comment/uncomment "console scripts", but still gets an error
